client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -216,7 +224,10 @@ class SimplePaymentsDialog extends Component {
 		}
 
 		productId
-			.then( id => this.props.onInsert( { id } ) )
+			.then( id => {
+				this.props.onInsert( { id } );
+				analytics.tracks.recordEvent( 'calypso_simple_payments_button_insert', { id } );
```

Comment:
And I think this can be updated to:
```
const mapDispatchToProps = {
    recordTracksEvent,
    //...
};
//...
this.props.recordTracksEvent( 'calypso_simple_payments_button_insert', { id } );
```